Resources/Locale/en-US/interaction/interaction-popup-component.ftl

```diff
@@ -42,4 +42,8 @@ comp-window-knock = *knock knock*
 
 hugging-success-generic = You hug {THE($target)}.
 hugging-success-generic-others = { CAPITALIZE(THE($user)) } hugs {THE($target)}.
-hugging-success-generic-target = { CAPITALIZE(THE($user)) } hugs you.
\ No newline at end of file
+hugging-success-generic-target = { CAPITALIZE(THE($user)) } hugs you.
+
+## No hands
+
+general-failure-no-hands = You don't have hands!
```

Comment:
I understand the confusion, but this ftl file is specifically for the interaction-popup component, which is used for things like hugging and petting. Its a component that exists explicitly to generate pop ups when interacting. This file is not for generic pop-ups triggered by all interactions.

This should probably be in `disposal-unit-component.ftl`

Reply:
I put on popup messages because i used popup for this message. Now its on disposal-unit-component.ftl again.